### DISCUSSION TITLE
articles/python-book-in-print

### DISCUSSION THREAD

**Discussion by giscus** [kaochenlong/kaochenlong-discuss#10] [General]
# articles/python-book-in-print

為了給讀者最準確的理解，作者花了一年多整理資料並深入研究 Python 原始碼，只為揭開 Python 中那些容易被忽略或誤解的細節。新手自學時常受到不正確觀念的影響，因此在書中力求呈現真正的運作原理。這本書不僅是一本程式自學書，更是引導您深入理解技術的指引幫助您掌握 Python 核心概念，為邁向專業奠定堅實基礎。

https://pythonbook.cc/articles/python-book-in-print?fbclid=IwY2xjawGn6O5leHRuA2FlbQIxMAABHWj4fUp3vmtYbEWCpJL7h5Z_UwzjsfH3dlhLhlnSZdEw5QzrH_5b35LyfA_aem_a22q0VSHebkTOsM0wAh5OQ

**Comment by cochiachang** [comment]
祝福暢銷大賣^^

**Comment by letter57** [comment]
謝謝龍哥無私的分享...也祝龍哥這本書大賣，就跟"為你自己學git"一樣...超喜歡龍哥學習的態度..

**Comment by stwc** [comment]
已買了一本支持！
想請問一下電子書檔名叫「read-ruby」是故意取的嗎XD

**Reply by kaochenlong** [reply]
應該不是故意的。

因為這個專案之前是打算用來擺放 Ruby 的書，我記得應該都改過來了，不知道為什麼還留著之前的名字，不過就當做是美麗的誤會也不錯 :)
